javascriptv3/example_code/lambda/README.md

```diff
@@ -1,30 +1,63 @@
-# AWS Lambda JavaScript SDK v3 code examples
-AWS Lambda lets you run code without provisioning or managing servers. You pay only for the compute time you consume.
+# AWS Lambda code examples for the AWS SDK for JavaScript in Node.js
+
+## Overview
+
+Shows how to use the SDK for JavaScript in Node.js with AWS Lambda (Lambda) to manage and invoke
+functions.
+
+AWS Lambda is a serverless, event-driven compute service that lets you run code for virtually any type of application or backend service without provisioning or managing servers. You can trigger Lambda from over 200 AWS services and software as a service (SaaS) applications, and only pay for what you use.
```

Comment:
(optional) I would keep just the first sentence of this description.

Reply:
Removed.